### ISSUE TITLE
Show the total number of messages

### ISSUE THREAD

**Issue by pcav** [k9mail/k-9#1037]
See this patch:

http://grokbase.com/t/gg/k-9-mail/137crp6hfg/total-number-of-messages-in-a-folder-account

I would find this very useful. Having only the number of unread messages is not enough for my use.
Anything preventing to merge it?

**Comment by cketti** [comment]
The linked message doesn't contain a patch.

Also, the interface is already displaying lots of information. Adding even more won't make the situation any better.
One day we might add a details screen that can show the total number of messages and maybe how much of your quota you're using. But that's a very low priority item.

**Comment by pcav** [comment]
You're right about the patch, sorry. The code might be available from the message author however.
As for the clutter, I do not agree: in my screen I have at top right the number of unread messages, with lots of empty space on its right. Adding unread/total (e.g. 3/49) would not clutter the interface, and to me at least would be very useful.
Thanks for your reply.
